app/src/ui/tab-bar.tsx

```diff
@@ -22,6 +26,9 @@ interface ITabBarProps {
 
   /** The type of TabBar controlling its style */
   readonly type?: TabBarType
+
+  /** Navigate via drag over */
+  readonly allowDragOverSwitching?: boolean
```

Comment:
Is there a reason to make this nullable? Seems like false might suffice as a default?

Reply:
Reason I have it optional is so I do not have to specify it 13 places it is used, and AFAIK you can't set a default value on an interface.